src/components/molecules/NavbarSchedule/NavbarSchedule.scss

```
@@ -0,0 +1,48 @@
+@import "scss/constants.scss";
+
+.nav-party-logo {
+  position: relative;
+  background-color: rgba($white, 0.08);
+  height: 60px;
+  padding: 0 50px 0 10px;
+  text-align: center;
+  cursor: pointer;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  transition: transform 400ms cubic-bezier(0.23, 1, 0.32, 1);
```

This has been extracted into a variable: https://github.com/sparkletown/sparkle/blob/staging/src/scss/constants.scss#L62

```suggestion
  transition: transform 400ms $transition-function;
```